pom.xml

```diff
@@ -27,7 +27,7 @@
 		<cloud.sdk.version>5.11.0</cloud.sdk.version>
```

Comment:
Could be Cloud SDK 5.12.0: https://sap.github.io/cloud-sdk/docs/java/release-notes#5120---september-03-2024